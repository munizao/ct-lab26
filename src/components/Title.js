import React from 'react';
import styles from './App.css';
import PropTypes from 'prop-types';

const Title = (props) => (
  <div>
    <h2>{props.name}</h2>
    <p>id: {props.id} -- created 2 years ago.</p>
  </div>
);

Title.propTypes = {
  name: PropTypes.string,
  id: PropTypes.number 
};

export default Title;
