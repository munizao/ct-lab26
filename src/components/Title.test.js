import React from 'react';
import { shallow } from 'enzyme';
import Title from './Title';

describe('Title component', () => {
  it('renders Title', () => {
    const wrapper = shallow(<Title id="132" name="Gar's Mytholog" />);
    expect(wrapper).toMatchSnapshot();
  });
});
