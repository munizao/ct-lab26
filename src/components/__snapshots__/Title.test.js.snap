// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component renders Title 1`] = `
<div>
  <h2>
    Gar\\'s Mytholog
  </h2>
  <p>
    id: 
    132
     -- created 2 years ago.
  </p>
</div>
`;
